src/components/POLine/Location/LocationView.js

```diff
@@ -23,14 +29,20 @@ const Location = ({ location, locationsMap, holdings }) => {
   const locationValue = location.holdingId
     ? getHoldingLocationName(holding, locationsMap)
     : locationNameCode;
+  const locationName = location.holdingId
+    ? `${fieldName}.holdingId`
```

Comment:
name not required (name: PropTypes.string,) here and we can get problem. Could you please investigate this problem and similar problem across this pull?

Reply:
Good catch! The absence of the 'name' prop will result in a confusing name being assigned. Thanks!